pyproject.toml

```diff
@@ -46,14 +46,15 @@ quality = [
   "flake8~=4.0.1",
   "isort~=5.10.1",
   "pre-commit~=2.20.0",
```

Comment:
let's talk about this! I'd not make it a hard dependency of the core tests, if possible.

I think we could have separate tests for the different model wrappers, like what `timemachines` does (we don't need to display the badge though).
https://github.com/microprediction/timemachines
The wrappers will break more frequently, but `core` tests should always pass.

Reply:
I can make this happen, by the way!

Reply:
Alright! I'm removing this for now.

Reply:
Also, I have put it in the wrong place, I wanted it under ``tests = []``.